iothub/device/src/IDelegatingHandler.cs

```diff
@@ -34,6 +34,10 @@ internal interface IDelegatingHandler : IContinuationProvider<IDelegatingHandler
 
         Task EnableReceiveMessageAsync(CancellationToken cancellationToken);
 
+        // This is to ensure that if device connects over MQTT with CleanSession flag set to false,
+        // then any message sent while the device was disconnected is delivered on the callback.
+        Task EnsurePendingMessagesAreDelivered(CancellationToken cancellationToken);
```

Comment:
Async suffix?

Reply:
we don't have any impl in here, just the signature